tests/docker_ssi/test_docker_ssi_crash.py

```diff
@@ -34,7 +34,7 @@ def setup_crash(self):
     @bug(condition=context.library not in ("python", "nodejs", "dotnet"), reason="INPLAT-11")
     @irrelevant(context.library == "python" and context.installed_language_runtime < "3.7.0")
```

Comment:
If possible, would like to sneak this in there too (given that we officially stopped supporting 3.7 after the major upgrade) 😅 
```suggestion
    @irrelevant(context.library == "python" and context.installed_language_runtime < "3.8.0")
```

Reply:
I'll do this in a separate PR